app/src/sharedTest/java/org/oppia/android/app/databinding/StateAssemblerMarginBindingAdaptersTest.kt

```diff
@@ -0,0 +1,455 @@
+      val textView: TextView = it.findViewById(R.id.test_margin_text_view)
+      setQuestionSplitViewMargins(textView)
+      return@runWithActivity textView
+    }
+    assertThat(textView.marginStart.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginStart()
+    )
+    assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginEnd()
+    )
+  }
+
+  @Test
+  fun testMarginBindingAdapters_questionSplitViewMargin_rtlIsEnabled_marginsForRtlIsCorrect() {
+    val textView = activityRule.scenario.runWithActivity {
+      val textView: TextView = it.findViewById(R.id.test_margin_text_view)
+      setQuestionSplitViewMargins(textView)
+      textView.layoutDirection = View.LAYOUT_DIRECTION_RTL
+      return@runWithActivity textView
+    }
+    assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginStart()
+    )
+    assertThat(textView.marginStart.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginEnd()
+    )
+  }
+
+  @Config(qualifiers = "port")
+  @Test
+  fun testMarginBindingAdapters_ltrIsEnabled_port_marginStartAndMarginEndForLtrIsCorrect() {
+    val textView = activityRule.scenario.runWithActivity {
+      val textView: TextView = it.findViewById(R.id.test_margin_text_view)
+      setExplorationViewMargins(textView)
+      return@runWithActivity textView
+    }
+    assertThat(textView.marginStart.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginStart()
+    )
+    assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginEnd()
+    )
+  }
+
+  @Config(qualifiers = "land")
+  @Test
+  fun testMarginBindingAdapters_ltrIsEnabled_landscape_marginStartAndMarginEndForLtrIsCorrect() {
+    val textView = activityRule.scenario.runWithActivity {
+      val textView: TextView = it.findViewById(R.id.test_margin_text_view)
+      setExplorationViewMargins(textView)
+      return@runWithActivity textView
+    }
+    assertThat(textView.marginStart.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginStart()
+    )
+    assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginEnd()
+    )
+  }
+
+  @Config(qualifiers = "sw600dp-port")
+  @Test
+  fun testMarginBindingAdapters_ltrEnabled__port_tablet_marginStartAndMarginEndForLtrIsCorrect() {
+    val textView = activityRule.scenario.runWithActivity {
+      val textView: TextView = it.findViewById(R.id.test_margin_text_view)
+      setExplorationViewMargins(textView)
+      return@runWithActivity textView
+    }
+    assertThat(textView.marginStart.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginStart()
+    )
+    assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(
+      getExplorationMarginEnd()
+    )
+  }
+
+  @Config(qualifiers = "sw600dp-land")
+  @Test
+  fun testMarginBindingAdapters_ltrEnabled_landscapeTablet_marginStartAndEndForLtrIsCorrect() {
```

Comment:
```suggestion
  fun testMarginBindingAdapters_ltrEnabled_landscape_tablet_marginStartAndEndForLtrIsCorrect() {
```

(for consistency with your test name above). Please make sure all new tests in each of the new test suites are consistent in this way.